src/App.test.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
+import {Provider} from 'react-redux';
```

Comment:
@Spetastian Can you make it so that these spaces between curlies are preserved?

Reply:
For imports

Reply:
Yes, I do agree. This is the reason for why we also need eslint aside for prettier. To. Specify our own rules as well.

Reply:
Yes, I do agree. This is the reason for why we also need eslint aside for prettier. To. Specify our own rules as well.